agent/src/main/java/com/code_intelligence/jazzer/runtime/BUILD.bazel

```diff
@@ -16,6 +16,18 @@ java_binary(
     ],
 )
 
+# These classes with public Bazel visibility are contained in jazzer_bootstrap.jar
+# and will thus be available on the bootstrap class path.
+java_binary(
+    name = "jazzer_bootstrap_env",
+    create_executable = False,
+    visibility = ["//driver/src/main/java/com/code_intelligence/jazzer:__pkg__"],
+    runtime_deps = [
+        "//agent/src/main/java/com/code_intelligence/jazzer/api:hooks",
```

Comment:
This would mean that these classes will be included in both `jazzer.jar` and `jazzer_bootstrap.jar`. Does this mean that duplicate classes might be loaded?

Reply:
I added a comment to explain the purpose of this target better: It is used as a `deploy_env` of the actual Jazzer binary precisely to prevent this issue ;-)

Reply:
Great, thanks for clarifying this!